src/java.base/share/classes/java/lang/foreign/MemorySegment.java

```diff
@@ -623,8 +623,7 @@ default MemorySegment copyFrom(MemorySegment src) {
 
     /**
      * Copy the contents of this memory segment into a new byte array.
-     * @param elementLayout the source element layout. If the byte order associated with the layout is
-     * different from the {@linkplain ByteOrder#nativeOrder native order}, a byte swap operation will be performed on each array element.
+     * @param byteLayout the source element layout.
```

Comment:
Not sure these changes are a necessary consequence of the parameter naming harmonization. That is, the layout does indicate an array element layout...